frontend/src/app/home/language-picker/language-picker.component.ts

```diff
@@ -10,16 +15,28 @@ import { StorageHandlerService } from '../../services/storage-handler.service';
   styleUrls: ['./language-picker.component.scss'],
 })
 export class LanguagePickerComponent {
+  /**
+   * Holds the supoorted languages code, display name and chosen flag.
+   */
   supportedLanguages = [
     { localeCode: 'en', displayName: 'English', isChosen: false },
     { localeCode: 'de', displayName: 'Duetsch', isChosen: false },
```

Comment:
Here is a typo, "Deutsch" ;)

Reply:
Oh yeah thanks for notifying. Will fix it.